src/interpolatePath.js

```diff
@@ -310,19 +310,30 @@ export function pathCommandsFromString(d) {
  *
  * @param {Object[]} aCommandsInput Array of path commands
  * @param {Object[]} bCommandsInput Array of path commands
- * @param {Function} excludeSegment a function that takes a start command object and
+ * @param {(Function|Object)} interpolateOptions
```

Comment:
Couldn't find any obvious answer going through the JSDoc docs but I was hoping to do something like this

```js
/**
 * A function that returns true if the segment should be excluded from splitting.
 * @typedef ExcludeSegmentFn
 * @type {Function}
 * @param {Object} startCommand
 * @param {Object} endCommand
 * @returns {Boolean} True when segment should be excluded
 */

/**
 * @typedef InterpolateOptions
 * @type {Object}
 * @property {ExcludeSegmentFn} excludeSegment
 * @property {Boolean} [snapEndsToInput=true] a boolean indicating whether end of 
 * input should be sourced from input argument or computed. 
 */

/**
 * ...
 * @param {(ExcludeSegmentFn|InterpolateOptions)} interpolateOptions
**/
```